src/systems/collision.rs

```diff
@@ -195,22 +202,18 @@ impl<'s> System<'s> for BulletCollisionSystem {
             // Currently, bullet can be fired only horizontally
             if let Some(collidee_horizontal) = &collidee.horizontal {
                 match collidee_horizontal.name.as_ref() {
-                    "Boundary" => {}
-                    _ => {
+                    "Bullet" => {
```

Comment:
This change means that the bullet impact animation will be shown only when a bullet collides with a bullet. Hence, we won't see the animation when bullet collides with other entities. I think, in place of `"Bullet"` here, it should be `_`. And for the case when the collidee is `"Boundary"` we need to skip the logic that shows the animation. The way it was being done earlier.

Reply:
Yes, you're absolutely right. I'll change it back.